src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -63,11 +67,12 @@
 @Slf4j
 public class HubMain {
 
-    private static final DateTime startTime = new DateTime();
-    private static AppProperties appProperties = new AppProperties(PropertiesLoader.getInstance());
-    private static SystemProperties systemProperties = new SystemProperties(PropertiesLoader.getInstance());
-    private static ZookeeperProperties zookeeperProperties = new ZookeeperProperties(PropertiesLoader.getInstance());
-    private final StorageBackend storageBackend = StorageBackend.valueOf(appProperties.getHubType());
+    private static DateTime startTime = new DateTime();
```

Comment:
Is there a reason this went from `final` to not?

Reply:
nope. its an oversight